.github/workflows/e2e-private-cluster-trigger.yaml

```diff
@@ -0,0 +1,18 @@
+name: E2EPrivateClusterTrigger
+on:
+  schedule:
+    - cron: '7 0 * * 2'
+jobs:
+  private-cluster-trigger:
+    permissions:
+      id-token: write # aws-actions/configure-aws-credentials@v4.0.1
+      statuses: write # ./.github/actions/commit-status/start
+    uses: ./.github/workflows/e2e.yaml
+    with:
+      suite: PrivateCluster
+      region: us-east-1
+      workflow_trigger: "private_cluster"
+      # Default to true unless using a workflow_dispatch
+      cleanup: true
+    secrets:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Will we be able to know if a test run is successful or not from a private cluster in GitHub? Will we need to send a message from inside the private cluster? If so, we should update the message that is send to slack for this workflow run

Reply:
I'm not seeing the updates to the slack message, are you still thinking we should make that change?

Reply:
I have updated that. That message change is part of the e2e.yaml.